packages/ui/Touchable/Drag.js

```diff
@@ -81,6 +81,17 @@ class Drag {
 		this.move(coords);
 	}
 
+	// This method will get the `onHold`, `onHoldEnd`, `onHoldPulse` props and update the existing `holdConfig` values.
+	updateProps = ({onHold, onHoldEnd, onHoldPulse}) => {
+		// `holdConfig` might not exists a gesture not in progress. Check if gesture exists before updating the references to the `holdConfig`
+		if (!this.holdConfig) return;
+
+		// This will update the `holdConfig` with the new value
+		this.holdConfig.onHoldPulse = onHoldPulse;
+		this.holdConfig.onHold = onHold;
+		this.holdConfig.onHoldEnd = onHoldEnd;
+	}
```

Comment:
Similarly, this should use `!this.isDragging()` and the drag-specific callbacks.